APIBacklog/documentation/APIBacklog.md

```diff
@@ -3,51 +3,61 @@
+| Home Devices Quality on Demand | Telefonica | 2022/09/14 | [Link](https://github.com/camaraproject/WorkingGroups/blob/main/APIBacklog/documentation/SupportingDocuments/API%20proposals/APIproposal_HomeDevicesQoD_Telefonica.md) | DTAG, KDDI, Telefonica |  Approved (2022/10/13) | [HomeDevicesQoD](https://github.com/camaraproject/HomeDevicesQoD) | home_devices_qod | [Link](https://github.com/camaraproject/HomeDevicesQoD/blob/main/documentation/API_documentation) | v0.2.0 | [YAML](https://github.com/camaraproject/HomeDevicesQoD/blob/main/code/API_definitions/home_devices_qod.yaml) |  N/A |  N/A |  N/A|
+| | |  | | | | | | | v0.1.0 | [YAML](https://github.com/camaraproject/HomeDevicesQoD/blob/release-0.1.0/code/API_definitions/home_devices_qod.yaml) |  N/A |  N/A |  N/A|<!---SIMSwap-->%%%%%%%%%%%%%%%%%%%%%
+| SIM Swap | DTAG | 2022/09/30 | [Link](https://github.com/camaraproject/WorkingGroups/blob/main/APIBacklog/documentation/SupportingDocuments/API%20proposals/APIproposal_Sim%20Swap_DeustcheTelekom.md) | Airtel, DTAG, GSMA-IDG, KDDI, Teelfonica | Approved (2022/10/13) | [SIMSwap](https://github.com/camaraproject/SimSwap) | sim_swap | [Link](https://github.com/camaraproject/SimSwap/tree/main/documentation) | v0.4.1 | [YAML](https://github.com/camaraproject/SimSwap/blob/main/code/API_definitions/sim_swap.yaml) | N/A | N/A | N/A|N/A|%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%<!---Number Verification-->%%%%%%%%%%%%%%%%%%%%%
+| Number Verification | DTAG| 2022/09/30 | [Link](https://github.com/camaraproject/WorkingGroups/blob/main/APIBacklog/documentation/SupportingDocuments/API%20proposals/APIproposal_%20NumberVerification_DeustcheTelekom.md) | Airtel, DTAG, Orange, Telefonica | Approved (2022/10/13) | [NumberVerification](https://github.com/camaraproject/NumberVerification/tree/main) | number_verification | [Link](https://github.com/camaraproject/NumberVerification) | v0.4.0-wip | [YAML](https://github.com/camaraproject/NumberVerification/blob/main/code/API_definitions/number_verification.yaml) | N/A | N/A | N/A|N/A|N/A|%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%<!---OTPValidation-->%%%%%%%%%%%%%%%%%%%%%
+| OTP Validation | DTAG| 2022/09/30 | [Link](https://github.com/camaraproject/WorkingGroups/blob/main/APIBacklog/documentation/SupportingDocuments/API%20proposals/APIproposal_NumberVerificationMS2FA_DeustcheTelekom.md) | Airtel, DTAG, GSMA-IDG, KDDI, Telefonica  | Approved (2022/10/13) | [OTPValidationAPI](https://github.com/camaraproject/OTPValidationAPI) | one-time-password-sms | [Link](https://github.com/camaraproject/OTPvalidationAPI/blob/main/documentation/API_documentation) | v0.5.0 | [YAML](https://github.com/camaraproject/OTPvalidationAPI/blob/main/code/API_definitions/one-time-password-sms.yaml) | N/A | N/A | N/A|N/A|%%%%%%%%%%%%%%%%<!---Carrier Billing CheckOut-->%%%%%%%%%%%%%%%%%%%%%
```

Comment:
Links are pointing to main, not the release branch (also wrong in the sub project README, I opened an issue there).
YAML has inline documentation.